src/main/java/meteordevelopment/meteorclient/mixin/CommandSuggestorMixin.java

```diff
@@ -71,4 +74,9 @@ public void onRefresh(CallbackInfo ci, String string, StringReader reader) {
         }
     }
 
+    @Inject(method = "render", at = @At(value = "HEAD"), cancellable = true)
+    public void render(MatrixStack matrices, int mouseX, int mouseY, CallbackInfo info) {
```

Comment:
Give this method a better name.
Something like `onRenderSuggestion` would be better.

Reply:
Should be done now

Reply:
Sorry for so many commits. I do it in browser since I didn't setup git yet.